app/initializers/responsive.js

```diff
@@ -1,2 +1,32 @@
-import responsive from 'ember-responsive/initializers/responsive';
-export default responsive;
+import Ember from 'ember';
+import config from '../config/environment';
```

Comment:
Not necessarily - the way you have it is definitely fine just wanted to call it out.